api/lib/domain/usecases/index.js

```diff
@@ -22,6 +21,7 @@ import * as certificationCpfService from '../../../src/certification/shared/doma
 import * as certificationAssessmentRepository from '../../../src/certification/shared/infrastructure/repositories/certification-assessment-repository.js';
 import * as certificationCandidateRepository from '../../../src/certification/shared/infrastructure/repositories/certification-candidate-repository.js';
 import * as certificationCenterRepository from '../../../src/certification/shared/infrastructure/repositories/certification-center-repository.js';
+import * as certificationChallengeLiveAlertRepository from '../../../src/certification/shared/infrastructure/repositories/certification-challenge-live-alert-repository.js';
```

Comment:
Il me semble qu'on fait le trajet inverse de ce qui est évoqué dans le nom du commit
On peut renommer le commit:
`♻️ api: Move live alert injection from session to shared `